.dockerfiles/docker-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+# Use the user mounted wildbook-ia code to if mounted at /code
```

Comment:
This is a great extension of our current design, thank you for adding this flexibility